monasca-alarms/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM alpine:3.5
+ARG MON_CLIENT_VERSION
+FROM monasca/client:${MON_CLIENT_VERSION}
 
 # To force a rebuild, pass --build-arg REBUILD="$(DATE)" when running
 # `docker build`
 ARG REBUILD=1
 
 ENV MONASCA_WAIT_FOR_API=true \
-  KEYSTONE_DEFAULTS_ENABLED=false
+    KEYSTONE_DEFAULTS_ENABLED=false
```

Comment:
I know you did not change it but this variable should be set to `true` like in all other containers and like in monasca-alarms readme...

Reply:
Ok, will look into it. Thx @matrixik

Reply:
@matrixik done